schema/Makefile

```diff
@@ -16,3 +16,6 @@ validate: validate.go
 	go get -d ./...
 	go build ./validate.go
 
+clean:
```

Comment:
> I don't think that is necessary. `make clean1 can be executed correctly.

So can `make validate`, with or without the `.PHONY` declaration.  `.PHONY` *keeps* the target working even if someone creates a `clean` file.  And while that is unlikely, I don't see why we shouldn't protect ourselves from it.